### PR TITLE
Adding remote disable mechanism for logging

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
@wmathurin It's too intrusive to add log levels at this time. I'll do that as a different PR on the next story, once I get the basic instrumentation in first.

File: libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java [hunk 1]
```diff
@@ -206,6 +210,21 @@ public void disableOrEnableLogging(boolean enabled) {
     }
 
     /**
+     * Updates the preferences of this library.
+     */
+    public void updateLoggingPrefs() {
+        final AdminSettingsManager settingsManager = new AdminSettingsManager();
+        final String enabled = settingsManager.getPref(SalesforceAnalyticsManager.ANALYTICS_ON_OFF_KEY, account);
+        if (!TextUtils.isEmpty(enabled)) {
+            if (FALSE.equals(enabled.trim().toLowerCase())) {
```
Comment (wmathurin) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java]: Could you just use Boolean.parseBolean(enabled) ?

File: libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java [hunk 2]
```diff
@@ -206,6 +210,21 @@ public void disableOrEnableLogging(boolean enabled) {
     }
 
     /**
+     * Updates the preferences of this library.
+     */
+    public void updateLoggingPrefs() {
+        final AdminSettingsManager settingsManager = new AdminSettingsManager();
+        final String enabled = settingsManager.getPref(SalesforceAnalyticsManager.ANALYTICS_ON_OFF_KEY, account);
+        if (!TextUtils.isEmpty(enabled)) {
```
Comment (wmathurin) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java]: So if the key is not found, is it enabled or not?

Comment (bhariharan) [inline libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java]: Yes - enabled is the default setting.

**Comment by urthakkar** [comment]
There us another stories for log levels and configuring them remotely.
Right now, let's just have On/off and then we can add the levels

**Comment by bhariharan** [comment]
@wmathurin Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/1262/commits/0182ef748182fafbd2cda4adfbd2d91b0de7318d).

**Pull request merged**